### ISSUE TITLE
Reward system for Site in the steamname

### ISSUE THREAD

**Issue by GreYzZ** [ztizzlegaming/CSGOWinBig#138]
Hello,
I saw on the CSGOWINBIG.COM site that there is a reward system for players who 
put "CSGOWINBIG.COM" in their name.
So can someone help me getting this working with my site.

Or will @ztizzlegaming release this update here on github?

Steam: http://steamcommunity.com/id/GreYzZ_DE/

**Comment by Zouspants** [comment]
im also wondering if this gonna be updated or not! :)

**Comment by andrewda** [comment]
I could add this and make a PR if people are still interested?

**Comment by GreYzZ** [comment]
@andrewda 
You are in contact with zittle?

**Comment by andrewda** [comment]
@GreYzZ Nope. I've emailed him a few times about why no pull requests are ever merged, but he never replies. This project seems kind of dead, as no new features are being added. I might clone this and continue to add more features if nothing more is added.

**Comment by GreYzZ** [comment]
@andrewda 
This would be great if you can do this I believe the community will switch to your thread and help you over there. But yes it seems like this is dead
